packages/appstore/src/components/modals/real-wallets-upgrade/components/ready-to-upgrade-wallets/test/ready-to-upgrade-wallets.spec.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import ReadyToUpgradeWallets from '../ready-to-upgrade-wallets';
+import { StoreProvider, mockStore } from '@deriv/stores';
+
+let mock = mockStore({});
+
+const checkContainerReadyToUpgradeWallets = () => {
+    const toggleCheckbox = jest.fn();
+    const wrapper = ({ children }: { children: JSX.Element }) => <StoreProvider store={mock}>{children}</StoreProvider>;
+
+    const { container } = render(<ReadyToUpgradeWallets is_eu={false} toggleCheckbox={toggleCheckbox} />, {
+        wrapper,
+    });
+    expect(container).toBeInTheDocument();
+};
+
+describe('ReadyToUpgradeWallets', () => {
+    beforeEach(() => {
+        mock = mockStore({});
+    });
```

Comment:
why do we need this beforeEach function if there's no mock changes happening in each test case?